app/src/main/res/layout/topic_revision_fragment.xml

```diff
@@ -12,7 +12,7 @@
   <FrameLayout
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@color/component_color_topic_revision_fragment_background_color">
+    android:background="@color/component_color_shared_topic_activity_background_color">
```

Comment:
You can remove the `FrameLayout` and directly use this background in RecyclerView.

Reply:
Done.